tox.ini

```
@@ -109,7 +109,7 @@ description = Run integration tests
 deps =
     boto3
     bs4
-    juju
+    juju==3.0.4
```

```suggestion
    juju>=3.0.4
```
Correct me if i've misunderstood, can this work like so? So that future versions can also work.

---

Python juju client > 3.0.4 has some compatibility problems, so pinning the Python juju client to 3.0.4 for now.